package.json

```diff
@@ -11,7 +11,7 @@
     "fixFormatting": "prettier . --write",
     "lint": "xo .",
     "build": "tsc",
-    "removeSidecar": "rimraf -g ./**/*.js ./**/*.d.ts"
+    "removeSidecar": "rimraf -g ./!(node_modules)/**/*.js ./!(node_modules)/**/*.js"
```

Comment:
It looks like we're passing the same argument to `rimraf` twice here - should one of the arguments target `.d.ts` files rather than `.js` files?

Reply:
Ah, my bad. I was testing this command in a glob checker and must have copy pasted the same command twice. But now I think about it, I might as well condense this glob a bit more.

This should do the same job, but also should capture sidecar files at the top level of the project with the second glob.

`"rimraf -g ./!(node_modules)/**/@(*.js|*.d.ts) ./@(*.js|*.d.ts)"`